modules/apps/blogs/blogs-web/src/main/java/com/liferay/blogs/web/internal/display/context/BlogEntriesDisplayContext.java

```diff
@@ -335,11 +334,7 @@ else if (Objects.equals(orderByCol, "relevance")) {
 					Stream<SearchResult> stream = searchResults.stream();
 
 					return stream.map(
-						this::_toBlogsEntryOptional
-					).filter(
-						Optional::isPresent
-					).map(
-						Optional::get
+						this::_toBlogsEntry
```

Comment:
@LisztLiszt Please also remove this `Steam<>` usage. Since it's in a lambda, it's not caught by our scanning, but when you see it in the class you modify, please try to remove it.

cc @slnn @JanisZhang @W2jx @yuhai